go.mod

```diff
@@ -3,8 +3,9 @@ module github.com/ExpediaGroup/flyte-slack
 go 1.14
 
 require (
-	github.com/HotelsDotCom/flyte-client v0.0.0-20191018093418-d4ecefe9ad35
+	github.com/HotelsDotCom/flyte-client v1.0.0
```

Comment:
can we migrate to this version and repo
https://github.com/ExpediaGroup/flyte-client/tree/v2.0.0